.jshintrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
if you are adding jshint validation you need to make sure the current code will pass.  based on what I am seeing in my ide it will not